recipes/ssht/all/test_package/conanfile.py

```diff
@@ -15,6 +15,6 @@ def build(self):
         cmake.build()
 
     def test(self):
-        if not tools.cross_building(self):
-            os.chdir("bin")
-            self.run(".%sexample" % os.sep)
+        if not tools.cross_building(self.settings):
+            bin_path = os.path.join("bin", "test_package")
```

Comment:
Sorry for my bad copy pasting

```suggestion
            bin_path = os.path.join("bin", "example")
```